### PR TITLE
chore(branding): Rebrand SDK from OpenWorld to ExpediaGroup & Merge Core

### PR DESCRIPTION
**Pull request by osama-salman99**
* [x] Code is up-to-date with the `main` branch.
* [x] You've successfully built and run the tests locally.
* [ ] There are new or updated unit tests validating the change.

### :pencil: Description
- Rebrand SDK from OpenWorld to ExpediaGroup

### :link: Related Issues
N/A

### CODE REVIEW

File: .github/workflows/generator-publish-artifact.yaml [hunk 1]
```diff
@@ -12,10 +12,12 @@ jobs:
...
     steps:
-      - uses: actions/download-artifact@v3
+      - uses: actions/checkout@v4
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
         with:
-          name: sdk
-          path: generator/sdk
+          java-version: '11'
+          distribution: 'adopt'
```
Comment (anssari1) [inline .github/workflows/generator-publish-artifact.yaml]: change to 'temurin'

File: .github/workflows/generator-verify.yaml [hunk 2]
```diff
@@ -42,7 +42,7 @@ jobs:
         with:
           java-version: '11'
           distribution: 'adopt'
```
Comment (anssari1) [inline .github/workflows/generator-verify.yaml]: Change to 'temurin'

File: .github/workflows/generator-verify.yaml [hunk 3]
```diff
@@ -42,7 +42,7 @@ jobs:
...
-      - name: Set up core
+      - name: Set up parent
```
Comment (anssari1) [inline .github/workflows/generator-verify.yaml]: Do we still need to set up the parent?

File: .github/workflows/generator-publish-artifact.yaml (hunk 1 above)
Comment (osama-salman99) [reply .github/workflows/generator-publish-artifact.yaml]: Done ✅

File: .github/workflows/generator-verify.yaml (hunk 2 above)
Comment (osama-salman99) [reply .github/workflows/generator-verify.yaml]: Done ✅

File: .github/workflows/generator-verify.yaml (hunk 3 above)
Comment (osama-salman99) [reply .github/workflows/generator-verify.yaml]: Nope, just removed it ✅

**Review by anssari1 — APPROVED** [review]

**Pull request merged**